src/main/java/io/fabric8/runsh/RunShLoader.java

```diff
@@ -12,6 +12,8 @@
 import java.util.Scanner;
 import java.util.Set;
 
+import org.apache.commons.lang3.SystemUtils;
```

Comment:
Could we avoid this dependency ? Actually, detecting whether we are running under Unix is or not can be done with out any extra dependency.

Dependency should be only added if there is a very good reason.